generate_bib.py

```
@@ -1,39 +1,100 @@
+                if year_match:
+                    year = year_match.group(0)
+
+            if title and year:
+                citations_data.append({
+                    "Title": title,
+                    "Authors": authors,
+                    "Publication": journal,
+                    "Year": int(year),
+                    "PubMedID": pubmed_id,
+                })
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)